src/lib/monsters.lua

```diff
@@ -1414,6 +1414,172 @@ monster_list = [
+        token = "z966q",
+        size = "L",
+        type_str = "Monstrosity",
+        cr = 3,
+        pp = 11,
+    ),
+    configure_monster(
+        name = "Marilith",
+        image_url = "https://www.dndbeyond.com/avatars/thumbnails/30781/628/1000/1000/638061931913755502.png",
+        token = "pwp5v",
+        size = "L",
+        type_str = "Fiend (demon)",
+        cr = 16,
+        pp = 13,
+    ),
+    configure_monster(
+        name = "Mastiff",
+        image_url = "https://www.dndbeyond.com/avatars/thumbnails/30849/339/1000/1000/638064501182707662.png",
+        token = "437m2",
+        size = "M",
+        type_str = "Beast",
+        cr = 0.125,
+        pp = 13,
+    ),
+    configure_monster(
+        name = "Medusa",
+        image_url = "https://www.dndbeyond.com/avatars/thumbnails/30833/355/1000/1000/638063862056716335.png",
+        token = "6uqqt",
+        size = "M",
+        type_str = "Monstrosity",
+        cr = 6,
+        pp = 14,
+    ),
+    configure_monster(
+        name = "Merfolk",
+        image_url = "https://www.dndbeyond.com/avatars/thumbnails/30833/374/1000/1000/638063862492555953.png",
+        token = "5mcny",
+        size = "M",
+        type_str = "Humanoid (merfolk)",
+        cr = 0.125,
+        pp = 12, 
+    ),
+    configure_monster(
+        name = "Merrow",
+        image_url = "https://www.dndbeyond.com/avatars/thumbnails/30833/389/1000/1000/638063862754985837.png",
+        token = "nr7bq",
+        size = "Large",
+        type_str = "Monstrosity",
+        cr = 2,
+        pp = 10,
+    ),
+    configure_monster(
+        name = "Mimic",
+        image_url = "https://www.dndbeyond.com/avatars/thumbnails/30833/408/1000/1000/638063863232165584.png",
+        token = "z9c86",
+        size = "M",
+        type_str = "Monstrosity (shapechanger)",
+        cr = 2,
+        pp = 11,
+    ),
+    configure_monster(
+        name = "Minotaur",
+        image_url = "https://www.dndbeyond.com/avatars/thumbnails/30833/457/1000/1000/638063864308085049.png",
+        token = "bbcyr",
+        size = "Large",
+        type_str = "Monstrosity",
+        cr = 3,
+        pp = 17,
+    ),
+    configure_monster(
+        name = "Minotaur Skeleton",
+        image_url = "https://www.dndbeyond.com/avatars/thumbnails/30835/858/1000/1000/638063922665935822.png",
+        token = "2vsre",
+        size = "L",
+        type_str = "Undead",
+        cr = 2,
+        pp = 9, 
+    ),
+    configure_monster(
+        name = "Mule",
+        size = "M",
+        type_str = "Beast",
+        cr = 0.125,
+        pp = 10,
+    ),
+    configure_monster(
+        name = "Mummy",
+        image_url = "https://www.dndbeyond.com/avatars/thumbnails/30833/586/1000/1000/638063867954193239.png",
+        token = "ctpkj",
+        size = "M",
+        type_str = "Undead",
+        cr = 3,
+        pp = 10,
+    ),
+    configure_monster(

```

Comment:
this is a really iconic monster let's put the mummy image so we have something to look at